tests/auth_tests/test_tokens.py

```diff
@@ -69,9 +69,34 @@ def test_token_with_different_secret(self):
         self.assertTrue(p0.check_token(user, tk0))
         # Create and check a token with the default secret.
         p1 = PasswordResetTokenGenerator()
-        self.assertEqual(p1.secret, settings.SECRET_KEY)
-        self.assertNotEqual(p1.secret, new_secret)
         tk1 = p1.make_token(user)
         # Tokens created with a different secret don't validate.
         self.assertFalse(p0.check_token(user, tk1))
         self.assertFalse(p1.check_token(user, tk0))
+
+    def test_verify_with_multiple_secrets(self):
+        user = User.objects.create_user('tokentestuser', 'test2@example.com', 'testpw')
+
+        p1 = PasswordResetTokenGenerator()
+        p1.secret = 'secret'
+
+        p2 = PasswordResetTokenGenerator()
+        p2.secrets = ['newsecret', 'secret']
+
+        tk1 = p1.make_token(user)
+        self.assertTrue(p1.check_token(user, tk1))
```

Comment:
Please use `assertIs` for boolean checks. https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style (see under `assertIs`).